### PR TITLE
Am bug fix

### PR DESCRIPTION
**Pull request by anama118118**

### CODE REVIEW

File: BYT/View Controllers/FoaasViewController.swift [hunk 1]
```diff
@@ -120,6 +124,20 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
         }
         self.foaasView.mainTextLabel.text = validFoaas.message
         self.foaasView.subtitleTextLabel.text = validFoaas.subtitle
+
+        DispatchQueue.main.async {
+            let valueForConstraintConstant = validFoaas.subtitle.characters.count
+            if self.foaasView.subtitleLabelConstraint.constant - CGFloat(Double(valueForConstraintConstant) * 1.5) < 16 {
```
Comment (spacedrabbit) [inline BYT/View Controllers/FoaasViewController.swift]: can this not be re-written as `(CGFloat(valueForConstraintConstant) * 1.5) < 16.0`

File: BYT/View Controllers/FoaasViewController.swift [hunk 2]
```diff
@@ -120,6 +124,20 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
...
+                self.foaasView.subtitleLabelConstraint.constant = 16.0
+                print(self.foaasView.subtitleLabelConstraint.constant - CGFloat(valueForConstraintConstant))
+                self.foaasView.layoutIfNeeded()
+            } else {
+                self.foaasView.subtitleLabelConstraint.constant = self.foaasView.subtitleLabelConstraint.constant - CGFloat(Double(valueForConstraintConstant) * 1.5)
+                print(self.foaasView.subtitleLabelConstraint.constant - CGFloat(Double(valueForConstraintConstant) * 1.5))
```
Comment (spacedrabbit) [inline BYT/View Controllers/FoaasViewController.swift]: same for these two lines

File: BYT/View Controllers/FoaasViewController.swift [hunk 3]
```diff
@@ -120,6 +124,20 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
         }
         self.foaasView.mainTextLabel.text = validFoaas.message
         self.foaasView.subtitleTextLabel.text = validFoaas.subtitle
+
```
Comment (spacedrabbit) [inline BYT/View Controllers/FoaasViewController.swift]: can you add a short comment on what this dispatch closure is doing? it's not clear to me

**Review by spacedrabbit — CHANGES_REQUESTED** [review]

File: BYT/View Controllers/FoaasViewController.swift [hunk 4]
```diff
@@ -41,6 +41,10 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
         makeRequest()
         updateSettingsMenu()
         
+        UIView.animate(withDuration: 2.0) {
```
Comment (spacedrabbit) [inline BYT/View Controllers/FoaasViewController.swift]: 2 seconds is way too long from a UX perspective. Shorten this to something like  <= 0.3

**Review by spacedrabbit — CHANGES_REQUESTED** [review]

File: BYT/View Controllers/FoaasViewController.swift (hunk 3 above)
Comment (anama118118) [reply BYT/View Controllers/FoaasViewController.swift]: Okay done!

File: BYT/View Controllers/FoaasViewController.swift (hunk 1 above)
Comment (anama118118) [reply BYT/View Controllers/FoaasViewController.swift]: Sorry for the confusing constant name. I just updated it. Please see newest commit.

File: BYT/View Controllers/FoaasViewController.swift (hunk 4 above)
Comment (anama118118) [reply BYT/View Controllers/FoaasViewController.swift]: Done!

**Review by spacedrabbit — APPROVED** [review]
